kong-0.2.0-1.rockspec

```diff
@@ -1,9 +1,9 @@
 package = "kong"
-version = "0.1.1beta-2"
+version = "0.2.0-1"
 supported_platforms = {"linux", "macosx"}
 source = {
   url = "git://github.com/Mashape/kong",
-  tag = "0.1.1beta-2"
```

Comment:
Every release before `1.0` is supposed to be subject to changes, we can keep the `beta` out, as described at http://semver.org/